app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/deployCloudFormationStackConfig.html

```diff
@@ -71,6 +65,38 @@ <h4>Template Configuration</h4>
       update-pipeline="ctrl.updatePipeline"
     >
     </stage-artifact-selector-delegate>
+    <div ng-if="ctrl.isChangeSet()">
```

Comment:
Can you add this as a React component and import it here?

Reply:
Hi @caseyhebebrand 

I've added the component, thanks for the hints 🙇